src/components/PrimaryNav/PrimaryNav.js

```diff
@@ -78,6 +80,43 @@ const PrimaryNav = () => {
         "ext-nav-open": extNavActive,
       })}
     >
+      {showSwitchModal ? (
+        <Modal
+          close={() => setShowSwitchModal(false)}
+          title={"Switch back to the old Juju GUI"}
+        >
+          <p>
+            We're sorry to see you go, we would really appreciate it if you
+            could take a moment to tell us why you prefer the old GUI using the
+            methods below.
+          </p>
+          <p>
+            <a
+              className="p-list__link"
```

Comment:
This class wont do anything outside of its parent block `p-list`. You don't need it or the display block in the scss. Just put both links in separate `p`

Reply:
Done